BackEnd/validation/validate_moderation_url.py

```diff
@@ -0,0 +1,14 @@
+from django.core.exceptions import ValidationError
+
+
+def validate_url(profile, timestamp):
+    url_timestamp = int(timestamp)
+    if url_timestamp < int(profile.status_updated_at.timestamp()):
+        if profile.status == "pending":
```

Comment:
Use Enum

Reply:
Updated with choices of Profile status field